### PR TITLE
Extract simplejson library to utilities

### PR DESCRIPTION
**Pull request by LJBabbage**
### What is the context of this PR?
Extract simplejson library to utilities

### How to review 
Make sure everything works as before and that nothing has been missed

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: app/data_models/session_store.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def _load(self):
             except ValueError:
                 pass
 
-            self.session_data = json.loads(
+            self.session_data = loads_json(
```
Comment (LJBabbage) [inline app/data_models/session_store.py:105]: A slight pain and why I include **kwargs in all functions in utilities

File: app/utilities/simplejson.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import simplejson as json
```
Comment (LJBabbage) [inline app/utilities/simplejson.py:1]: Couldn't decide what to call this

File: app/utilities/simplejson.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+def load_json(file, **kwargs):
+    return json.load(file, **kwargs)
+
+
+def loads_json(data, **kwargs):
+    return json.loads(data, use_decimal=True, **kwargs)
+
+
+def dumps_json(data, **kwargs):
```
Comment (ajmaddaford) [inline app/utilities/simplejson.py:12]: I think the naming would read better as `json_load`, `json_loads` and `json_dumps`.

Comment (LJBabbage) [reply app/utilities/simplejson.py:12]: I took it from your suggestion in the other PR

File: app/utilities/simplejson.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    return json.dumps(data, for_json=True, **kwargs)
```
[the remark this answers is not in the document]
Comment (LJBabbage) [reply app/utilities/simplejson.py:13]: It wasn't on the original dump, only the loads

File: app/utilities/simplejson.py (hunk 2 above)
Comment (LJBabbage) [reply app/utilities/simplejson.py:1]: It's what I had original, but felt it might be too generic.

**Review by MebinAbraham — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**